relayer/src/utils.ts

```diff
@@ -29,24 +31,34 @@ export const isRelevantRecord =
             );
         };
 
-export const createParachainsMap = async (target: Target, configParachains: {
-    url: string,
-    paraId: number,
-    chain: string,
-}[]): Promise<ParachainsMap> => {
-    const map = new Map();
 
-    for (const { url, chain, paraId } of configParachains) {
-        const feedId = await target.sendCreateFeedTx();
-        const parachain = new Parachain({
-            feedId,
-            url,
-            chain: chain as ChainName,
-            logger
-        });
+type ParachainsMap = Map<string, Parachain>;
+
+export const createParachainsMap = async (
+    target: Target, 
+    configParachains: ParachainConfigType[],
+    signers: AddressOrPair[],
+): Promise<ParachainsMap> => {
+    const map = new Map();
 
+    for (let index = 0; index < configParachains.length; index++) {
```

Comment:
Use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/entries instead of manual indexing, it is less error-prone and I think more readable.

Reply:
but we need index to assign signer (on the next line)

Reply:
`.entries()` gives you both the element and its index.